swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -246,8 +247,82 @@ Follow the steps below to obtain API tokens for Google Sheets.
 1. Get the `clientID` and `ClientSecret` by following the [guidelines](https://developers.google.com/identity/protocols/oauth2).
 2. Follow the steps below to add https://developers.google.com/oauthplayground if you want to use OAuth 2.0 playground to receive the authorization code, access token, and refresh token.
 
-   1. Go to the OAuth 2.0 playground and click **Settings** on the top right-hand side.
-   2. Select **Use your own OAuth credentials** and provide the client ID and client secret obtained.
-   3. For step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use.
-   4. Proceed with the necessary Google account validation steps.
-   5. Click **Exchange authorization code for tokens** to obtain the refresh and access tokens.
\ No newline at end of file
+    1. Go to the OAuth 2.0 playground and click **Settings** on the top right-hand side. 
+    2. Select **Use your own OAuth credentials** and provide the client ID and client secret obtained. 
+    3. For step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use. 
+    4. Proceed with the necessary Google account validation steps. 
+    5. Click **Exchange authorization code for tokens** to obtain the refresh and access tokens.
+
+## Redis [Experimental]
+
+The Redis data store is a key-value-based inmemory datastore useful for storing and retrieving unstructured data quickly. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
+
+```shell
+$ bal persist init
+```
```

Comment:
```suggestion
The Redis data store is a key-value-based in-memory datastore useful for quickly storing and retrieving unstructured data. It is not the default data store for `bal persist`. Therefore, you need to specify the data store explicitly when generating the bal persist client APIs for Redis DB.

Below are the ways you can provide the data store for each client generation option

1. Integrate the client code generation with the package build.
```shell
$ bal persist add --datastore redis
\```

2. One-time generation of the client code against the data model.
```shell
$ bal persist generate --datastore redis
\```
```

Reply:
remove the \ at the end of command snippet

Reply:
Can you update this in other data stores?

Reply:
I have update this for all data stores in https://github.com/ballerina-platform/ballerina-dev-website/pull/8838/commits/c58208f496cbc3a70a6a1004d543b8de8bb8eed9